RoddyCore/src/de/dkfz/roddy/plugins/LibrariesFactory.groovy

```diff
@@ -211,6 +211,15 @@ public class LibrariesFactory extends Initializable {
         };
     }
 
+    static boolean checkFile(File f) {
+        return f.exists() && f.isFile() && f.canRead()
+    }
+
+    static boolean checkDirectory(File f) {
+        return f.exists() && f.isDirectory() && f.canRead()
+        //&& f.canExecute()
+    }
+
```

Comment:
Why are these functions here? Should they not be in the FileSystemAccessProvider? Maybe better put them into the RuntimeService until we have separated out the local and remote FSAPs, because LibFac does not contain any such functions yet and it is better to not add another place to fix.